src/App.css

```diff
@@ -1,24 +1,51 @@
+body {
+  background-color: #fafafa;
+}
+
 nav.navbar {
   display: flex;
   justify-content: flex-start;
+  background-color: #fff;
+  margin-bottom: 30px;
 }
 
 .nav-elements {
   display: flex;
   justify-content: space-evenly;
   align-items: baseline;
+  margin-left: 10px;
+}
+
+h1.site-title {
+  color: #0290ff;
+}
+
+a.navlink {
+  color: rgb(140, 140, 140);
+  text-decoration: none;
+  margin-left: 15px;
+}
+
+.navlink :hover {
+  color: black;
 }
 
 .item {
   display: flex;
   flex-direction: row;
   justify-content: space-evenly;
+  width: 80%;
+  background-color: #fff;
+  margin: 10px 0 10px 0;
+  padding-bottom: 15px;
 }
 
 .item-left {
   display: flex;
   flex-direction: column;
   justify-content: space-evenly;
+  width: 250px;
+  max-width: 250px;
 }
 
 .book-details {
```

Comment:
- [ ] As mentioned by the previous reviewer, can we please stick to the design provided in Zeplin? and don't worry if you find some stuff that is not relevant for this project like the comment or edit buttons, you can just add them as placeholders (without any functionality). see the difference below 👇 

## Ziplin
![image](https://user-images.githubusercontent.com/55465133/135727090-ad154f8f-00af-4b80-af15-a26aaf9f3330.png) 

## Your design
![image](https://user-images.githubusercontent.com/55465133/135727120-fb904197-2a84-4193-b0fa-b1e6d5d955c3.png)

As you can see there is still a difference, even in fonts. Can you please fix this issue?

- [ ] [OPTIONAL] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉